examples/RobotSimulator/HelloBulletRobotics.cpp

```diff
@@ -18,19 +18,21 @@ int main(int argc, char* argv[])
 	}
 
 	//remove all existing objects (if any)
-	sim->resetSimulation();
+	// sim->resetSimulation();
+	sim->resetSimulation(RESET_USE_REDUCED_DEFORMABLE_WORLD);
 	sim->setGravity(btVector3(0, 0, -9.8));
 	sim->setNumSolverIterations(100);
 	b3RobotSimulatorSetPhysicsEngineParameters args;
 	sim->getPhysicsEngineParameters(args);
 
 	int planeUid = sim->loadURDF("plane.urdf");
 	printf("planeUid = %d\n", planeUid);
+	
+	int r2d2Uid = sim->loadURDF("reduced_cube/reduced_cube.urdf");
```

Comment:
please revert this file or exclude it from this PR.

Reply:
will do